jitsi-utils-kotlin/src/test/kotlin/org/jitsi/utils/queue/QueueObserverBenchmarkTest.kt

```diff
@@ -199,7 +199,7 @@ class QueueObserverBenchmarkTest {
         var sumSquares: Long = 0
         for (duration in experimentDuration) {
             val diff = Math.abs(duration.toNanos() - averageNanos)
-            sumSquares = diff * diff
+            sumSquares += diff * diff
```

Comment:
stddev and avg calculation seems like a good candidate for a MathUtils class? (or if we don't mind being a little bit more invasive bring Apache Commons Math to the project)?